editor/filesystem_dock.cpp

```diff
@@ -2049,6 +2056,15 @@ Vector<String> FileSystemDock::_tree_get_selected(bool remove_self_inclusion, bo
 	return selected_strings;
 }
 
+Vector<String> FileSystemDock::_file_list_get_selected() const {
+	Vector<String> selected;
+
+	for (const int idx : files->get_selected_items()) {
```

Comment:
```suggestion
	for (int idx : files->get_selected_items()) {
```
We don't normally add `const` to simple types like this